runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -45,7 +45,7 @@ processResources {
   filter org.apache.tools.ant.filters.ReplaceTokens, tokens: [
     'dataflow.legacy_environment_major_version' : '8',
     'dataflow.fnapi_environment_major_version' : '8',
-    'dataflow.container_version' : 'beam-master-20210429',
+    'dataflow.container_version' : 'beam-master-20210503',
```

Comment:
It appears this image doesn't exist. Did you mean beam-master-20210503? (Or possibly it got auto deleted?)

Reply:
Yes I meant 20210505: https://github.com/apache/beam/pull/14744

Reply:
Sorry about that